js/components/harvest/job.vue

```diff
@@ -93,44 +87,43 @@ export default {
                 sort: 'status',
                 type: 'label',
                 width: 100,
-                label_type: function(status) {
-                    return STATUS_CLASSES[status];
-                },
-                label_func: function(status) {
-                    return STATUS_I18N[status];
-                }
-            }]
+                label_type: (status) =>  STATUS_CLASSES[status],
+                label_func: (status) => STATUS_I18N[status],
+            }],
+            p: new PageList({data: this.job.items}),
```

Comment:
The computed in this case was not working: the widgets were binded on the first instance and cannot detect deep changes